.github/ISSUE_TEMPLATE.md

```diff
@@ -8,7 +8,11 @@ This template is for bug reports. If you are reporting a bug, please continue on
 Note that leaving sections blank will make it difficult for us to troubleshoot and we may have to close the issue.
 -->
 
-**What version of ESLint are you using?**
+**Tell us about your environment**
+
+* **ESLint Version:**
+* **Node Version:**
+* **npm Version:**
```

Comment:
I don't see a downside to asking for this info. The fewer times we need to ask for more info, the faster triage goes.